File: repeated element in array/repeated.py
#Find the first repeating element in an array of integers

def firstrepeat(arr):
    obj={}
    for _ in arr:
        if _ in obj:
            return _
        else:
            obj[_]=1
    return "No repeating element"
arr=[10, 5, 3, 4, 3, 5, 6]
print(firstrepeat(arr))

File: repeated element in array/repeated.js
//Find the first repeating element in an array of integers
function firstrepeat(arr){
    let obj={};
    for (let i=0;i<arr.length;i++){
        if (obj[arr[i]]){
            return arr[i];
        }
        else{
            obj[arr[i]]=1;
        }
    }
    return "No repeating element";
}
console.log(firstrepeat([1,2,3,4,5,6,7,8,9,1]));